recipes/dspsr/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "dspsr" %}
+{% set version = "2022-02-17" %}
```

Comment:
I think you need to swap the hyphens for periods in your version string. Otherwise it will confuse the conda package resolution.

https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#info-index-json

Reply:
Yes, the Jinja trickery is already there, a few lines down:
```
    version: "{{ version | replace('-','.') }}"
```